pcs_test/tier0/test_lint.py

```diff
@@ -0,0 +1,40 @@
+import os
+import re
+from unittest import TestCase
+
+PACKAGE_DIR = os.path.realpath(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+)
+
+
+class IterableStr(TestCase):
+    maxDiff = None
+
+    def test_no_iterable_str_in_code(self):
+        paths_to_fix = []
+        for dirpath, dirnames, filenames in os.walk(
+            os.path.join(PACKAGE_DIR, "pcs")
+        ):
+            del dirnames
+            for fname in filenames:
+                if not fname.endswith(".py"):
+                    continue
+                file_path = os.path.join(dirpath, fname)
+                with open(file_path) as file:
+                    filedata = file.read()
+                    if re.search(
+                        r"\b(collection|iterable|sequence)\[[^][]*str",
```

Comment:
maybe this way it will be easier to read `\b(collection|iterable|sequence)\[[^\[\]]*str`